### ISSUE TITLE
Bug in value2Interface(): reflect.Bytes() called on Array value

### ISSUE THREAD

**Issue by teejays** [go-xorm/xorm#1345]
I noticed that in [value2Interface function in session_convert.go](https://github.com/go-xorm/xorm/blob/master/session_convert.go#L645), we are using `reflect.Value.Bytes()` function when the fieldType is `reflect.Array` OR `reflect.Slice`. However, go's `reflect.Value.Bytes()` only works when the value is of type `reflect.Slice`, and panics otherwise.

I would propose that change the _if_ condition from:
```
if (k == reflect.Array || k == reflect.Slice) && (...) {
    // logic
}
```
to
```
if (k == reflect.Slice) && (...) {
    // logic
}
```

**Comment by lunny** [comment]
@teejays could you send a PR to fix that? Thanks.

**Comment by teejays** [comment]
Thanks @lunny, I created a PR: https://github.com/go-xorm/xorm/pull/1346

**Comment by lunny** [comment]
Closed by #1346
